.changeset/beige-rocks-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-migrator': minor
+---
+
+Add new migration scss-replace-zero-values
```

Comment:
TODO: Potentially update this changelog entry depending on final scope of migration